packages/generator/src/v1.1/backcompat/generateOntologyRuntimeDistDir.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2023 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as path from "node:path";
+import type { MinimalFs } from "../../MinimalFs";
+import { formatTs } from "../../util/test/formatTs";
+import { generateAggregationsDir } from "./internal-foundry-ontology-runtime-dist/generateAggregationsDir";
+import { generateOntologyProviderDir } from "./internal-foundry-ontology-runtime-dist/generateOntologyProviderDir";
+
+export async function generateOntologyRuntimeDistDir(
+  outDir: string,
+  fs: MinimalFs,
+) {
+  const runtimeDistDir = path.join(
+    outDir,
+    "internal",
+    "@foundry",
+    "ontology-runtime",
+    "dist",
+  );
+  await fs.mkdir(runtimeDistDir, { recursive: true });
+
+  await generateOntologyProviderDir(fs, runtimeDistDir);
+  await generateAggregationsDir(fs, runtimeDistDir);
+
+  await fs.writeFile(
+    path.join(runtimeDistDir, "index.ts"),
+    // TRASHFIXME
```

Comment:
Do you intend to remove these markers and clean up the commented out code?

Reply:
The markers were for you or me in the future. This does not complete the job